Rakefile

```diff
@@ -84,6 +84,32 @@ task :console do
   IRB.start
 end
 
+desc "Run acceptance tests"
+task :acceptance do |version, pe_version|
+  version ||= '1.9.2.0'
+  pe_version ||= '2018.1'
+  cmd = <<HEREDOC
+export BUNDLE_PATH=.bundle/gems
+export BUNDLE_BIN=.bundle/bin
+# We should change this to SHA and SUITE_VERSION in the acceptance tests
+export PE_RAZOR_SERVER_PACKAGE_BUILD_VERSION=#{version}
+
+bundle install
+
+beaker-hostgenerator --pe_dir=http://enterprise.delivery.puppetlabs.net/#{pe_version}/ci-ready \
```

Comment:
We use beaker-hostgen in https://github.com/puppetlabs/ci-job-configs/blob/master/resources/scripts/vanagon-integration.sh#L69 -- I think we can parametrize it to include additional options to account for Razor. We should probably remove it here in the rake task, or at least create a separate one that just runs beaker (since I think having a rake task to run stuff locally could also be valuable).

Reply:
Also, we'd want this rake task in the acceptance/ folder.